db/models/app.model.js

```diff
@@ -11,5 +8,15 @@ exports.selectAllTopics = () => {
   });
 };
 
+exports.selectArticleById = (article_id) => {
+  const paramValues = [];
+  const query = "SELECT * FROM articles WHERE article_id = $1";
+  paramValues.push(article_id);
 
-
+  return db.query(query, paramValues).then(({ rows }) => {
+    if (!rows.length) {
+      return Promise.reject({ status: 404, msg: "Not Found" });
+    }
+    return rows;
```

Comment:
With regards to my first comment: *Here* you should return rows[0], as we would only ever expect 1 article to come back, it doesn't need to be an array